### DISCUSSION TITLE
Reviewers

### DISCUSSION THREAD

**Discussion by Tommylovespython** [community/community#44343] [New to GitHub] [answered]
Hi I’m new here. Was wondering why do people add reviewers/collaborators to their repos. Is that safe?

**Comment by jge162** [comment, marked as answer]
Reviewers can add a second eye to your code, they are not hopefully going to ruin your program. But I think it’s safe. Since u most likely will know the reviewer. Hope that makes sense.

**Reply by Tommylovespython** [reply]
After that explanation I think I do see a benefit. Thanks.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community @Tommylovespython - great question and appreciate you answering with your perspective @jge162 ⭐
